clap_builder/src/builder/command.rs

```diff
@@ -4729,7 +4734,7 @@ impl Command {
                     .help("Print help (see more with '--help')")
                     .long_help("Print help (see a summary with '-h')");
             } else {
```

Comment:
This is an example of one case where we have removed hardcoded English text and replaced it with a template string containing a text lookup key.